crates/stdx/src/thread/pool.rs

```diff
@@ -86,6 +86,7 @@ impl Pool {
         self.job_sender.send(job).unwrap();
     }
 
+    #[allow(clippy::len_without_is_empty)]
```

Comment:
And this is why we don't really run clippy on the code base 😅 the defaults are sometimes questionable

Reply:
It's not Clippy's fault, I'd expect `len` to return either the thread count or the pending queue size, not the number of running jobs.

Reply:
Overall yes, I do want us to run clippy. I just don't like some of the default set of lints it has. Related https://github.com/rust-lang/rust-analyzer/issues/15918